generator/src/lib/premadeProducts.ts

```diff
@@ -20,4 +20,14 @@ export const premadeProducts = [
     product: "177389-09633",
     id: "steel-roll-1234b",
   },
+  {
+    name: "Sandvik Salt Battery Z37-310-76 75kWh (TBA)",
+    product: "fzsonick-sodium-nickel-chloride",
+    id: "TBA",
```

Comment:
`TBA` is not an ID, `Z37-310-76` seems to be a battery model from fzsonick, not sandvik, so this has to be wrong.